<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1887</width>
    <height>881</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>WenQuanYi Micro Hei Mono</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>470</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="humidity_label1">
        <property name="font">
         <font>
          <family>WenQuanYi Micro Hei Mono</family>
          <pointsize>19</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>TEMPERATURE(*C)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="humidity_lcdNumber1">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="digitCount">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>501</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>550</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>WenQuanYi Micro Hei</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Time(Sec)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="time_lcdNumber"/>
    </item>
   </layout>
  </widget>
  <widget class="QCustomPlot" name="plotA" native="true">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>511</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="start_pushButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>640</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="humidity_lcdNumber">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>WenQuanYi Micro Hei Mono</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="digitCount">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLabel" name="humidity_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>470</y>
     <width>299</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>28</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Relative Humidity(%)</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>511</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QCustomPlot" name="plot_temp" native="true">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>40</y>
     <width>551</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>470</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>RESISTANCE(ohms)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="temperature_lcdNumber">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
